repl/src/main/scala/com/cloudera/livy/repl/Session.scala

```diff
@@ -168,4 +195,21 @@ class Session(interpreter: Interpreter, stateChangedCallback: SessionState => Un
 
     compact(render(resultInJson))
   }
+
+  private def setJobGroup(statementId: Int): String = {
+    val cmd = Kind(interpreter.kind) match {
+      case Spark() | PySpark() | PySpark3() =>
+        s"""sc.setJobGroup("$statementId", "Job group for statement $statementId")"""
```

Comment:
> But since Scala 2.11 doesn't provide res variables, I think we can call sc.setJobGroup("$statementId", "Job group for statement $statementId") in Scala 2.11.

Can you explain more, I cannot get your point.